corporate/views/support.py

```diff
@@ -602,16 +614,23 @@ def remote_servers_support(
                     context["error_message"] = error.msg
 
     email_to_search = None
+    uuid_to_search = None
     hostname_to_search = None
     if query:
         search_text = query.strip()
         if "@" in search_text:
             email_to_search = search_text
         else:
-            hostname_to_search = search_text
+            try:
+                uuid.UUID(search_text, version=4)
```

Comment:
We could alternatively use a regex here to check the string.

Reply:
Regex might be better in that in server logs, we sometimes only have the first ~14 characters of a UUID. Not sure, we can see what feels useful in practice.